src/store/moviesSlice.js

```diff
@@ -36,6 +64,22 @@ export const moviesSlice = createSlice({
       .addCase(fetchMovies.rejected, (state, action) => {
         state.error = action.error;
         state.status = 'failed';
+      })
+      // TODO: handle errors for add/edit/delete to show them on UI
+      .addCase(addMovie.fulfilled, (state, action) => {
+        state.movies.push(action.payload);
+        // TODO: refetch movies because of sort and filter
```

Comment:
One of the many ways of handling this could be to add a flag in the `movies` state (something like `shouldRefetcMovies`) and then watch for that flag in the `MovieList` component by using `useEffect` to get the latest response

Reply:
Done